diff --git a/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts b/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-ax1800-pro.dts
similarity index 94%
rename from target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts
rename to target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-ax1800-pro.dts
index b53ae071b..e6272abfb 100644
--- a/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-re-ss-01.dts
+++ b/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-ax1800-pro.dts
@@ -11,14 +11,14 @@
 
 / {
 	model = "JDCloud RE-SS-01 (AX1800 Pro)";
-	compatible = "jdcloud,re-ss-01", "qcom,ipq6018";
+	compatible = "jdcloud,ax1800-pro", "qcom,ipq6018";
 
 	aliases {
 		serial0 = &blsp1_uart3;
-		led-boot = &led_status_green;
+		led-boot = &led_status_red;
 		led-failsafe = &led_status_red;
-		led-running = &led_status_blue;
-		led-upgrade = &led_status_red;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_green;
 
 		ethernet1 = "/soc/dp2";
 		ethernet2 = "/soc/dp3";

