diff --git a/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network b/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network
index 7d8c3ab54..5e421ed54 100644
--- a/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network
+++ b/target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network
@@ -11,27 +11,22 @@ ipq60xx_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
-	anysafe,e1|\
 	cmiot,ax18|\
 	jdcloud,re-cs-07|\
-	jdcloud,re-ss-01|\
-	philips,ly1800|\
+	jdcloud,ax1800-pro|\
 	qihoo,360v6|\
 	redmi,ax5-*|\
 	xiaomi,rm1800)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3" "wan"
 		;;
-	dptech,ap3000-2c|\
-	glinet,gl-axt1800|\
-	link,nn6000-v1)
-		ucidef_set_interfaces_lan_wan "lan1 lan2" "wan"
-		;;
 	glinet,gl-ax1800|\
 	jdcloud,re-cs-02|\
-	link,nn6000-v2|\
 	linksys,mr7350)
 		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
 		;;
+	glinet,gl-axt1800)
+		ucidef_set_interfaces_lan_wan "lan1 lan2" "wan"
+		;;
 	*)
 		echo "Unsupported hardware. Network interfaces not initialized"
 		;;
diff --git a/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh b/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh
index c820e5217..b6378f532 100644
--- a/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh
+++ b/target/linux/qualcommax/ipq60xx/base-files/lib/upgrade/platform.sh
@@ -10,7 +10,6 @@ platform_check_image() {
 
 platform_do_upgrade() {
 	case "$(board_name)" in
-	anysafe,e1|\
 	cmiot,ax18|\
 	glinet,gl-ax1800|\
 	glinet,gl-axt1800|\
@@ -20,10 +19,7 @@ platform_do_upgrade() {
 		;;
 	jdcloud,re-cs-02|\
 	jdcloud,re-cs-07|\
-	jdcloud,re-ss-01|\
-	link,nn6000-v1|\
-	link,nn6000-v2|\
-	philips,ly1800|\
+	jdcloud,ax1800-pro|\
 	redmi,ax5-jdcloud)
 		kernelname="0:HLOS"
 		rootfsname="rootfs"
diff --git a/target/linux/qualcommax/image/ipq60xx.mk b/target/linux/qualcommax/image/ipq60xx.mk
index 4be530e91..1f68735a3 100644
--- a/target/linux/qualcommax/image/ipq60xx.mk
+++ b/target/linux/qualcommax/image/ipq60xx.mk
@@ -1,24 +1,9 @@
 define Device/EmmcImage
-	IMAGES += factory.bin recovery.bin
-	IMAGE/factory.bin := append-kernel | pad-to 12288k | append-rootfs | append-metadata
-	IMAGE/recovery.bin := append-kernel | pad-to 6144k | append-rootfs | append-metadata
+	IMAGES := factory.bin sysupgrade.bin
+	IMAGE/factory.bin := append-kernel | pad-to 6144k | append-rootfs | append-metadata
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to 64k | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
-define Device/anysafe_e1
-	$(call Device/FitImage)
-	$(call Device/UbiFit)
-	DEVICE_VENDOR := AnySafe
-	DEVICE_MODEL := E1
-	BLOCKSIZE := 128k
-	PAGESIZE := 2048
-	DEVICE_DTS := ipq6010-anysafe-e1
-	DEVICE_DTS_CONFIG := config@cp01-c3
-	DEVICE_PACKAGES := ath11k-firmware-qcn9074 ipq-wifi-anysafe_e1 \
-		kmod-ath11k-pci kmod-hwmon-pwmfan
-endef
-TARGET_DEVICES += anysafe_e1
-
 define Device/cmiot_ax18
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
@@ -31,42 +16,39 @@ define Device/cmiot_ax18
 endef
 TARGET_DEVICES += cmiot_ax18
 
-define Device/dptech_ap3000-2c
+define Device/glinet_gl-common
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
-	DEVICE_VENDOR := DPtech
-	DEVICE_MODEL := AP3000-2C
+	DEVICE_VENDOR := GL.iNet
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@cp03-c1
 	SOC := ipq6000
-	DEVICE_DTS_CONFIG := config@cp03-c1-DP019
-	DEVICE_PACKAGES := ipq-wifi-dptech_ap3000-2c
 endef
-TARGET_DEVICES += dptech_ap3000-2c
 
-define Device/glinet_gl-common
+define Device/glinet_gl-ax1800
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
 	DEVICE_VENDOR := GL.iNet
+	DEVICE_MODEL := GL-AX1800
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
 	DEVICE_DTS_CONFIG := config@cp03-c1
 	SOC := ipq6000
-endef
-
-define Device/glinet_gl-ax1800
-	$(call Device/glinet_gl-common)
-	DEVICE_MODEL := GL-AX1800
 	DEVICE_PACKAGES := ipq-wifi-glinet_gl-ax1800
-	SUPPORTED_DEVICES += glinet,ax1800
 endef
 TARGET_DEVICES += glinet_gl-ax1800
 
 define Device/glinet_gl-axt1800
-	$(call Device/glinet_gl-common)
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := GL.iNet
 	DEVICE_MODEL := GL-AXT1800
-	DEVICE_PACKAGES := ipq-wifi-glinet_gl-axt1800 kmod-hwmon-pwmfan
-	SUPPORTED_DEVICES += glinet,axt1800
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@cp03-c1
+	SOC := ipq6000
+	DEVICE_PACKAGES := ipq-wifi-glinet_gl-axt1800 kmod-hwmon-gpiofan
 endef
 TARGET_DEVICES += glinet_gl-axt1800
 
@@ -79,7 +61,7 @@ define Device/jdcloud_re-cs-02
 	BLOCKSIZE := 64k
 	KERNEL_SIZE := 6144k
 	DEVICE_DTS_CONFIG := config@cp03-c3
-	DEVICE_PACKAGES := ath11k-firmware-qcn9074 ipq-wifi-jdcloud_re-cs-02 kmod-ath11k-pci
+	DEVICE_PACKAGES := ipq-wifi-jdcloud_re-cs-02 kmod-ath11k-pci ath11k-firmware-qcn9074
 endef
 TARGET_DEVICES += jdcloud_re-cs-02
 
@@ -96,7 +78,7 @@ define Device/jdcloud_re-cs-07
 endef
 TARGET_DEVICES += jdcloud_re-cs-07
 
-define Device/jdcloud_re-ss-01
+define Device/jdcloud_ax1800-pro
 	$(call Device/FitImage)
 	$(call Device/EmmcImage)
 	DEVICE_VENDOR := JDCloud
@@ -105,28 +87,9 @@ define Device/jdcloud_re-ss-01
 	BLOCKSIZE := 64k
 	KERNEL_SIZE := 6144k
 	DEVICE_DTS_CONFIG := config@cp03-c2
-	DEVICE_PACKAGES := ipq-wifi-jdcloud_re-ss-01
-endef
-TARGET_DEVICES += jdcloud_re-ss-01
-
-define Device/link_nn6000-v1
-	$(call Device/FitImage)
-	$(call Device/EmmcImage)
-	DEVICE_VENDOR := Link
-	DEVICE_MODEL := NN6000 v1
-	SOC := ipq6000
-	KERNEL_SIZE := 6144k
-	DEVICE_DTS_CONFIG := config@cp03-c1
-	DEVICE_PACKAGES := ipq-wifi-link_nn6000 kmod-fs-f2fs f2fs-tools
-	IMAGE/factory.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-rootfs | append-metadata
+	DEVICE_PACKAGES := ipq-wifi-jdcloud_ax1800pro
 endef
-TARGET_DEVICES += link_nn6000-v1
-
-define Device/link_nn6000-v2
-	$(Device/link_nn6000-v1)
-	DEVICE_MODEL := NN6000 v2
-endef
-TARGET_DEVICES += link_nn6000-v2
+TARGET_DEVICES += jdcloud_ax1800-pro
 
 define Device/linksys_mr7350
 	$(call Device/FitImage)
@@ -145,19 +108,6 @@ define Device/linksys_mr7350
 endef
 TARGET_DEVICES += linksys_mr7350
 
-define Device/philips_ly1800
-	$(call Device/FitImage)
-	$(call Device/EmmcImage)
-	DEVICE_VENDOR := Philips
-	DEVICE_MODEL := LY1800
-	SOC := ipq6010
-	BLOCKSIZE := 64k
-	KERNEL_SIZE := 6144k
-	DEVICE_DTS_CONFIG := config@cp01-c1
-	DEVICE_PACKAGES := ipq-wifi-philips_ly1800
-endef
-TARGET_DEVICES += philips_ly1800
-
 define Device/qihoo_360v6
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
